### PR TITLE
Added exception for ConnectionError after sysprep is rebooting the instance during scenario building.

### PR DESCRIPTION
**Pull request by AlexandruTudose**
@alexandrucoman

### CODE REVIEW

**Comment by alexcoman** [comment]
The https://github.com/cloudbase/cloudbase-init-ci/pull/6/commits/2abef3c502cf531e80f38b28765f7c44bd406d55 commit is not bond to your GitHub account, please fix that.

File: argus/recipes/cloud/windows.py [hunk 1]
```diff
@@ -238,9 +239,10 @@ def sysprep(self):
         try:
             self._backend.remote_client.run_command(
                 'powershell C:\\sysprep.ps1')
-        except (socket.error, winrm_exceptions.WinRMTransportError):
-            # This error is to be expected because the vm will restart
-            # before sysprep.ps1 finishes execution.
+        except (socket.error, winrm_exceptions.WinRMTransportError, requests.ConnectionError):
+            # At this point after sysprep the instance is rebooted.
```
Comment (alexcoman) [inline argus/recipes/cloud/windows.py]: This comment block causes more confusion please rephrase it.

**Comment by AlexandruTudose** [comment]
@alexandrucoman  it is ok now?

**Comment by alexcoman** [comment]
No, your pull requests contain a lot of modified commits from the repo history. You should rebase over the master branch and fix all the issues that may come out.

**Comment by alexcoman** [comment]
Please remove the dot from the end of the commit message.

**Comment by AlexandruTudose** [comment]
@alexandrucoman I successfully tested it on server 2016 and also on windows 10 and there are no longer failing scenarios because of connectivity issues.

**Pull request merged**